core/src/main/java/org/arquillian/smart/testing/configuration/Range.java

```diff
@@ -0,0 +1,24 @@
+package org.arquillian.smart.testing.configuration;
+
+public class Range {
```

Comment:
Why do we have a separated class for the `Range`? What is the difference between having `Scm` + `Range` and only `ScmRange`?

Reply:
The main idea to keep both classes separated to keep same structure in system properties  (`scm.range.head` & `scm.range.tail`) & in yaml
```yaml
scm:
    range:
      head: HEAD
      tail: HEAD~3
```

Otherwise we have to give option in config file as 
```yaml
scmRange:
      head: HEAD
      tail: HEAD~3
```

Also for this second option, we need to think about `lastChanges` option, as it's defined as `scm.lastChanges` as system prioperty for now. So either we need to change it to `scmRange.lastChanges` if we decided to go with second option